Misc/NEWS.d/next/Core and Builtins/2023-01-23-22-06-07.gh-issue-98831.FkCDuV.rst

```diff
@@ -0,0 +1,2 @@
+Rewrite the implementation of :opcode:`CHECK_EG_MATCH` in the instruction
+definition DSL.
```

Comment:
Actually I've not been adding news for anything related to the generator, assuming nobody cares about its progress.